.ci/pipeline_definitions

```diff
@@ -17,13 +21,12 @@ autoscaler:
                 build: ~
             registry: 'gcr-readwrite'
             image: 'eu.gcr.io/gardener-project/gardener/autoscaler/cluster-autoscaler'
-            dockerfile: 'Dockerfile.amd64'
-            dir: 'cluster-autoscaler'
+            dockerfile: './cluster-autoscaler/Dockerfile'
```

Comment:
If you write it this way https://github.com/gardener/autoscaler/blob/4d7787f19559533c5f2e9a0aef710b9000e84523/.ci/pipeline_definitions#L20-L21
Then I think our publish step problem would be resolved.

Reply:
Then docker would not be able to copy the `.ci/build` script into the build container. It does not allow copying files from parent directories into containers.

Reply:
oh okay, then I think no changes required.